ports/docopt/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "docopt",
-  "version-date": "2022-03-15",
+  "version-date": "2024-04-11",
```

```suggestion
  "version-date": "2022-03-15",
  "port-version": 1,
```

The port's version should not get bumped because you are not updating docopt.

---

but then ./vcpkg x-add-version --all does not work. There was a "force" argument mentioned, should I use this?

---

Yes, you should revert your changes in `versions/` first.

---

To clarify: You should not change `version-date` . `"port-version": 1` is a necessary addition.

---

Please add `"port-version": 1` to vcpkg.json

---

Like this? Sorry, I really do not get this process..